test/ui/JobsRolesTests.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env mocha */
+/* global browser */
+const webdriver = require('selenium-webdriver');
+const chai = require('chai');  
+
+describe('View Job Roles Flow', async () => {
+    
+    /*
+    UI Test For View Job Roles
+
+    Navigate to the job roles list
+
+    Expect the page should display a valid list of job roles fetched from the /job-roles API endpoint.
+    */
+    it('Displays a list of job roles on the job roles page', async () => {
+
+        var driver = new webdriver.Builder().
+            withCapabilities(webdriver.Capabilities.chrome()).
+            build();
+
+        // Visit the job roles page
+        await driver.get("http://localhost:3000/jobs")
```

Comment:
don't hardcode localhost, pass in as env var so we could run these against any URL

Reply:
Changed to using env variable